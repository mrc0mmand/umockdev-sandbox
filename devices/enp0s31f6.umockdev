P: /devices/pci0000:00/0000:00:1f.6/net/enp0s31f6
E: INTERFACE=enp0s31f6
E: IFINDEX=2
E: SUBSYSTEM=net
E: ID_NET_NAMING_SCHEME=v247
E: ID_NET_NAME_MAC=enx8c8caa458e68
E: ID_OUI_FROM_DATABASE=LCFC(HeFei) Electronics Technology co., ltd
E: ID_NET_NAME_PATH=enp0s31f6
E: ID_BUS=pci
E: ID_VENDOR_ID=0x8086
E: ID_MODEL_ID=0x0d4e
E: ID_PCI_CLASS_FROM_DATABASE=Network controller
E: ID_PCI_SUBCLASS_FROM_DATABASE=Ethernet controller
E: ID_VENDOR_FROM_DATABASE=Intel Corporation
E: ID_MODEL_FROM_DATABASE=Ethernet Connection (10) I219-LM
E: ID_MM_CANDIDATE=1
E: ID_PATH=pci-0000:00:1f.6
E: ID_PATH_TAG=pci-0000_00_1f_6
E: ID_NET_DRIVER=e1000e
E: ID_NET_LINK_FILE=/usr/lib/systemd/network/99-default.link
E: ID_NET_NAME=enp0s31f6
E: SYSTEMD_ALIAS=/sys/subsystem/net/devices/enp0s31f6
E: TAGS=:systemd:
E: CURRENT_TAGS=:systemd:
A: addr_assign_type=0\n
A: addr_len=6\n
A: address=8c:8c:aa:45:8e:68\n
A: broadcast=ff:ff:ff:ff:ff:ff\n
A: carrier=1\n
A: carrier_changes=6\n
A: carrier_down_count=3\n
A: carrier_up_count=3\n
A: dev_id=0x0\n
A: dev_port=0\n
L: device=../../../0000:00:1f.6
A: dormant=0\n
A: duplex=full\n
A: flags=0x1003\n
A: gro_flush_timeout=0\n
A: ifalias=
A: ifindex=2\n
A: iflink=2\n
A: link_mode=0\n
A: mtu=1500\n
A: name_assign_type=4\n
A: napi_defer_hard_irqs=0\n
A: netdev_group=0\n
A: operstate=up\n
A: power/control=auto\n
A: power/runtime_active_time=0\n
A: power/runtime_status=unsupported\n
A: power/runtime_suspended_time=0\n
A: proto_down=0\n
A: speed=100\n
A: statistics/collisions=0\n
A: statistics/multicast=1984505\n
A: statistics/rx_bytes=98751134011\n
A: statistics/rx_compressed=0\n
A: statistics/rx_crc_errors=0\n
A: statistics/rx_dropped=1\n
A: statistics/rx_errors=0\n
A: statistics/rx_fifo_errors=0\n
A: statistics/rx_frame_errors=0\n
A: statistics/rx_length_errors=0\n
A: statistics/rx_missed_errors=0\n
A: statistics/rx_nohandler=0\n
A: statistics/rx_over_errors=0\n
A: statistics/rx_packets=80675242\n
A: statistics/tx_aborted_errors=0\n
A: statistics/tx_bytes=4665018409\n
A: statistics/tx_carrier_errors=0\n
A: statistics/tx_compressed=0\n
A: statistics/tx_dropped=0\n
A: statistics/tx_errors=0\n
A: statistics/tx_fifo_errors=0\n
A: statistics/tx_heartbeat_errors=0\n
A: statistics/tx_packets=43121489\n
A: statistics/tx_window_errors=0\n
A: testing=0\n
A: threaded=0\n
A: tx_queue_len=1000\n
A: type=1\n

P: /devices/pci0000:00/0000:00:1f.6
E: DRIVER=e1000e
E: PCI_CLASS=20000
E: PCI_ID=8086:0D4E
E: PCI_SUBSYS_ID=17AA:22AF
E: PCI_SLOT_NAME=0000:00:1f.6
E: MODALIAS=pci:v00008086d00000D4Esv000017AAsd000022AFbc02sc00i00
E: SUBSYSTEM=pci
E: ID_PCI_CLASS_FROM_DATABASE=Network controller
E: ID_PCI_SUBCLASS_FROM_DATABASE=Ethernet controller
E: ID_VENDOR_FROM_DATABASE=Intel Corporation
E: ID_MODEL_FROM_DATABASE=Ethernet Connection (10) I219-LM
A: ari_enabled=0\n
A: broken_parity_status=0\n
A: class=0x020000\n
H: config=86804E0D060410000000000200000000000070EB000000000000000000000000000000000000000000000000AA17AF2200000000C800000000000000FF010000
A: consistent_dma_mask_bits=64\n
A: d3cold_allowed=1\n
A: device=0x0d4e\n
A: dma_mask_bits=64\n
L: driver=../../../bus/pci/drivers/e1000e
A: driver_override=(null)\n
A: enable=1\n
L: firmware_node=../../LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:1b
A: irq=143\n
A: local_cpulist=0-7\n
A: local_cpus=ff\n
A: modalias=pci:v00008086d00000D4Esv000017AAsd000022AFbc02sc00i00\n
A: msi_bus=1\n
A: msi_irqs/143=msi\n
A: numa_node=-1\n
A: power/control=on\n
A: power/runtime_active_time=2831822425\n
A: power/runtime_status=active\n
A: power/runtime_suspended_time=0\n
A: power/wakeup=enabled\n
A: power/wakeup_abort_count=0\n
A: power/wakeup_active=0\n
A: power/wakeup_active_count=25\n
A: power/wakeup_count=0\n
A: power/wakeup_expire_count=25\n
A: power/wakeup_last_time_ms=1562000619\n
A: power/wakeup_max_time_ms=117\n
A: power/wakeup_total_time_ms=2634\n
A: power_state=D0\n
A: resource=0x00000000eb700000 0x00000000eb71ffff 0x0000000000040200\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n
A: revision=0x00\n
A: subsystem_device=0x22af\n
A: subsystem_vendor=0x17aa\n
A: vendor=0x8086\n

