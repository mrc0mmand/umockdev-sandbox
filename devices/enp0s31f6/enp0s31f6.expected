ID_NET_NAME_MAC=enx8c8caa458e68
ID_OUI_FROM_DATABASE=LCFC(HeFei) Electronics Technology co., ltd
ID_NET_NAME_PATH=enp0s31f6
